src/main/webapp/new/app/shared/domain/game-map/node.ts

```diff
@@ -149,6 +149,17 @@ export class Node {
         }
     }
 
+    isJoint() {
+        let hexesCount = 0;
+        for (let i in this.hexes) {
+            hexesCount++;
+            if (hexesCount > 1) {
+                return true;
+            }
+        }
+        return false;
```

Comment:
maybe we can replace this method with something like this: 
`return this.hexes.size > 0`